browser/ui/speedreader/speedreader_bubble_browsertest.cc

```diff
@@ -55,12 +55,19 @@ IN_PROC_BROWSER_TEST_F(SpeedreaderBubbleBrowserTest,
   ShowAndVerifyUi();
 }
 
+#if defined(OS_MAC)
+// Disable this test for intermittent crashes.
+// See https://github.com/brave/brave-browser/issues/20082
+IN_PROC_BROWSER_TEST_F(SpeedreaderBubbleBrowserTest,
+                       DISABLED_InvokeUi_speedreader_mode_bubble_basic) {
```

Comment:
the [Chromium way](https://source.chromium.org/search?q=%22%23define%20MAYBE_%22%20filepath:%5C.*test.cc&sq=&ss=chromium) is to define the test name as `MAYBE_` and change it via `#define`:
```
#if defined(OS_MAC)
#define MAYBE_InvokeUi_speedreader_mode_bubble_basic \
  DISABLED_InvokeUi_speedreader_mode_bubble_basic
#else
#define MAYBE_InvokeUi_speedreader_mode_bubble_basic \
  InvokeUi_speedreader_mode_bubble_basic
#endif
IN_PROC_BROWSER_TEST_F(BrowserKeyEventsTest,
                       MAYBE_InvokeUi_speedreader_mode_bubble_basic) {
```

Reply:
Thanks! I've updated the style.